src/jdk.hotspot.agent/linux/native/libsaproc/symtab.c

```diff
@@ -307,7 +306,6 @@ static struct symtab* build_symtab_internal(int fd, const char *filename, bool t
   ELF_SHDR* shbuf = NULL;
   ELF_SHDR* cursct = NULL;
   ELF_PHDR* phbuf = NULL;
```

Comment:
phbuf is also essentially unused. The only reference is to free it if non-null, but it's always NULL, so that code can be removed too.

Reply:
Good catch! Also removed similar code from macOS version.